client/src/containers/header/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from 'react';
+import { Header as HeaderComponent } from 'components';
+import * as actions from '../user/logic/actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { IHeaderProps } from 'components/common/header/index';
+import menuProfile from 'assets/icons/header/menu/avatar.svg';
+import menuSettings from 'assets/icons/header/menu/settings.svg';
+import menuChallenge from 'assets/icons/header/menu/goal.svg';
+import menuSignout from 'assets/icons/header/menu/logout.svg';
+import { ROUTES } from 'constants/routes';
+import { useHistory } from 'react-router-dom';
+import { IRootState } from 'typings/root-state';
+
+
+
+const Header = () => {
+    const history = useHistory();
+    const dispatch = useDispatch();
+    const handleLogout = async () => {
+        dispatch(actions.logoutUser());
+    };
+    const user = useSelector((state: IRootState) => state.auth.userData.user);
```

Comment:
better to use our custom hook 'useAppSelector'

Reply:
thank you, fixed